pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -148,7 +146,7 @@ func (fr *FileReader) storeDashboardsInFoldersFromFileStructure(filesFoundOnDisk
 
 		folderID, err := getOrCreateFolderID(fr.Cfg, fr.dashboardProvisioningService, folderName)
 		if err != nil && err != ErrFolderNameMissing {
-			return err
+			return fmt.Errorf("can't provision %s folder from files structure: %w", folderName, err)
```

Comment:
I think it's better to quote the folder name in case it's got whitespace, and it should probably be "file system structure":
```suggestion
			return fmt.Errorf("can't provision folder %q from file system structure: %w", folderName, err)
```

Reply:
Yep, file, you are right. 
As for quote marks, I tried it because you had mentioned it in one of my previous PRs, and quote marks were looking like this.
```
EROR[08-03|23:33:23] A service failed                         logger=server err="Failed to provision dashboards: Failed to provision config default: can't provision \"General\" folder from files structure: A folder with that name already exists"
```
But maybe it's better to use them instead of thinking about spaces.

Reply:
@nabokihms Yeah double quotes will get escaped in this logging format, it's no big deal. It's better than being confused by spaces :)